addons/barcodes_gs1_nomenclature/models/barcode_rule.py

```diff
@@ -22,8 +22,8 @@ def _default_encoding(self):
             ('package', 'Package'),
             ('use_date', 'Best before Date'),
             ('expiration_date', 'Expiration Date'),
-            ('package_type', 'Packaging Type'),
-            ('packaging_date', 'Packaging Date'),
+            ('package_type', 'Package Type'),
+            ('package_date', 'Package Date'),
```

Comment:
Have we to change that ?
For now, it's here but it's even not yet implemented in `stock_barcode`. Also, I understand the confusion with package/packaging but `pack_date`/"Pack Date" doesn't help IMO because with this solution, we don't know neither if it 
refers to a package or a product's packaging.

"Packaging Date" is the right term regarding GS1 specs, but the Odoo usage would be the package's date. For now, there is no field for that, but it'll be soon (cf. https://github.com/odoo/odoo/pull/82389/commits/2ce9dc32ca357f3c05681a24860c18325de5bef0#diff-9a22420c4a72cc9b6d558b3d832846c2e44a4f3731c6df9579d66c6279c4804eR1061) and its name will be `packaging_date`. So even if it's confusing right now, I find it's even more confusing to change it.

(I'm talking only about the packaging date, I think you're right to rename "Packaging Type" to "Package Type")

Reply:
Ah, the `packaging_date` name in 2ce9dc3 isn't set in stone. It can be changed depending on what we decide is best here. Also "Pack Date" matches the "Put in Pack" term we already use.

Reply:
Ah you right, I didn't take "Put in Pack" in the equation but that's true... I give up ! Choosing a name is a too stressful activity and programmers shouldn't have to deal with that because it's a common knowledge we are bad for that :fearful: 

More seriously, I think Packaging Date is a OK name for the field and for the GS1 rule's type so it doesn't need to be changed, but if you guys want to change it, I'm OK with that too.